addons/hr_timesheet/migrations/11.0.1.0/post-migration.py

```diff
@@ -42,6 +42,9 @@ def recompute_tasks_from_issues_fields(env):
     being inserted by SQL, don't have these fields computed, so we force here
     its recomputation.
     """
+    if not openupgrade.column_exists(
+            env.cr, 'account_analytic_line', 'issue_id'):
```

Comment:
Please use `origin_issue_column`

Reply:
The PR works well when issue modules have not been installed. The `if` statement is copied from the method `migrate_project_issue_sheet` just above. Feel free to change it if desired. I don't know how to use `origin_issue_column`.

Reply:
@norlinhenrik `origin_issue_column` it's a variable just below. It's because the method is touching that column, not the other, so for consistency, you should check for that column.